src/app/patient-dashboard/common/formentry/formentry.component.ts

```diff
@@ -279,14 +276,15 @@ export class FormentryComponent implements OnInit, OnDestroy {
         this.router.navigate(['/patient-dashboard/patient/' +
           this.patient.uuid + '/general/general/program-manager/new-program', 'step', 3]);
         break;
-      case 'oncologyReferral':
+      case 'programReferral':
```

Comment:
If we don't have another type of referral conflicting `patientReferral` then I propose we move everything to be renamed `patientReferral`
```suggestion
      case 'patientReferral':
```